obspy/clients/eida/__init__.py

```diff
@@ -10,120 +13,182 @@
 :license:
     GNU Lesser General Public License, Version 3
     (https://www.gnu.org/copyleft/lesser.html)
+
+Basic Usage
+-----------
+
+The first step is always to initialize a client object.
+
+>>> from obspy.clients.eida import Client
+>>> client = Client("GFZ")
+
+A client object can be initialized either with the base URL of any EIDA routing
+service or with a shortcut name that will be mapped to such a URL. The examples
```

Comment:
> HTTP error is the only thing that is sure. I cannot tell if routing is really not supported on purpose of if the node is temporarily mis-configured. Could check application.wadl, but that is not really reliable either (eg., Apache gives 503 for 60 seconds each time when the service is restarted). obspy.clients.eida could have its own URL_MAPPINGS of nodes that run the routing service. Then we would have to do a pull request each time a node is added or removed. In practice there would be rarely a need to use another node than "GFZ" (default), because all instances of the EIDA routing service are synchronized and use the same set of nodes.

OK, should be fine as is I think. Probably it would just add confusion to have a separate dictionary for URL mappings.